turborepo-tests/example-with-yarn/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "turborepo-examples-test-with-yarn",
```

Comment:
These used to be prefixed with `@turborepo-examples-tests/` , fwiw. I'd generally prefer to keep behavior and style changes in separate commits in main (i.e. separate PRs, since we squash and merge). If you're changing it anyway, how about `@turbo-test/example-with-yarn`, etc.?

Reply:
Mistake on my part. I'll keep that prefix such that the workspace name is only dropping the package manager.